specifications/config.json

```diff
@@ -0,0 +1,127 @@
+{
+  "client": {
+    "apiKey":{
+      "file":"",
+      "id":"",
+      "secret":""
+    },
+    "cacheManager":{
+      "defaultTtl": 300,
+      "defaultTti": 300,
+      "caches": {
+        "account": {
+          "ttl": 300,
+          "tti": 300
+        }
+      }
+    },
+    "baseUrl": "https://api.stormpath.com/v1",
+    "connectionTimeout": 30,
+    "authenticationScheme": "SAUTHC1", // or "BASIC"
+    "proxy":{
+      "port": "",
+      "host": "",
+      "username": "",
+      "password": ""
+    }
+  },
+  "application": {
+    "name": "test",
+    "href": "https://..."
+  },
+  "web": {
+    "oauth2":{
+      "enabled": false,
+      "uri": "/oauth/token",
+      "password":{
+        "enabled": true,
+        "accessToken":{
+          "ttl": 3600
+        }
+      },
+      "client_credentials":{
+        "enabled": true,
+        "accessToken":{
+          "ttl": 3600
+        }
+      }
+    },
+    "accessTokenCookie":{
+      "name": "access_token",
+      "ttl": 3600,
+      "tti": 900,
+      "httpOnly": true,
+      "secure": null,
+      "path": "/",
+      "domain": null
+    },
+    "register": {
+      "enabled": false,
+      "uri": "/register",
+      "nextUri": "/",
+      "autoAuthorize": false,
+      "fields": [
```

Comment:
So.. After working on this with @robertjd, we think this may be a bad idea to use an array for fields.

EG: What happens if you want to only disable a specific field or two? Here's the use case we bumped into that was causing trouble:

We want to disable JUST the given name.

In this case, we'd have to say:

fields: [ { name: 'givenName', required: false }, { name: 'surname', required: true }, ... ]

That's a ton of typing ;(